mycroft/enclosure/gui.py

```diff
@@ -349,6 +350,67 @@ def show_url(self, url, override_idle=None,
         self.show_page("SYSTEM_UrlFrame.qml", override_idle,
                        override_animations)
 
+    def register_settings(self):
+        """Register requested skill settings
+        configuration in GUI.
+
+        Registers handler to apply settings when
+        updated via GUI interface.
+        Register handler to update settings when
+        updated via Web interface.
+        """
+        skill_id = self.skill.skill_id
+        settingmeta_path = join(self.skill.root_dir,
+                                "settingsmeta.json")
+        self.settings_gui_generator.populate(skill_id,
+                                             settingmeta_path,
+                                             self.skill.settings)
+        apply_handler = skill_id + ".settings.set"
+        update_handler = skill_id + ".settings.update"
+        self.skill.gui.register_handler(apply_handler,
+                                        self.skill.gui.apply_settings)
+        self.skill.gui.register_handler(update_handler,
+                                        self.skill.gui.update_settings)
+
+    def display_settings(self, override_idle=True,
```

Comment:
Should this be named `show_settings` to match the other show_x methods

Reply:
fixed, renamed to show_settings